string bag_filepath
---
ElasticFusionOutput elastic_fusion_output